lib/plugins/acts_as_journalized/lib/acts_as_journalized.rb

```diff
@@ -96,44 +89,11 @@ def acts_as_journalized(options = {}, &block)
           (journal_hash[:except] ||= []) << primary_key << inheritance_column <<
             :updated_on << :updated_at << :lock_version << :lft << :rgt
 
-          prepare_journaled_options(journal_hash)
+          journal_hash = prepare_journaled_options(journal_hash)
```

Comment:
Lets see what happens :)

I could leave out most of the changes here, since the greatest part is only refactoring:
* finding out that great parts are not actually being used and can therefore be deleted
* changing a method to not modify its input (this line)

If you think this is a bigger problem I can also not do those changes...